### PR TITLE
Fs1352 req new link

### PR DESCRIPTION
**Pull request by srh-sloan**
Passes additional information into the notification service:
- URL to request a new magic link
- Fund name - now retrieved from fund-store
- Fund contact email - now retrieved from fund-store

Also ran pre-commit so there are some formatting changes too

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -4,10 +4,10 @@ FROM python:3.10-bullseye
 WORKDIR /app
 COPY requirements.txt requirements.txt
 # Install git to download utils library
-RUN apt update && apt -yq install git 
+RUN apt update && apt -yq install git
```
Comment (gidsg) [inline Dockerfile:7]: looks like just whitespace changes can we revert?

File: Dockerfile [hunk 2]
```diff
@@ -4,10 +4,10 @@ FROM python:3.10-bullseye
...
 RUN python3 -m pip install --upgrade pip && pip install -r requirements.txt
 COPY . .
 
 EXPOSE 8080
 
-CMD ["flask", "run", "--host", "0.0.0.0", "--port", "8080"]
\ No newline at end of file
+CMD ["flask", "run", "--host", "0.0.0.0", "--port", "8080"]
```
Comment (gidsg) [inline Dockerfile:13]: looks like just whitespace changes can we revert?

File: api/magic_links/routes.py [hunk 3]
```diff
@@ -15,7 +15,6 @@
 
 
 class MagicLinksView(MagicLinkMethods, MethodView):
-
```
Comment (gidsg) [inline api/magic_links/routes.py:18]: Looks like just whitespace, can we revert?

File: config/envs/default.py [hunk 4]
```diff
@@ -93,6 +95,18 @@ class DefaultConfig(object):
...
+        CommonConfig.NOTIFICATION_MAGIC_LINK_CONTACT_HELP_EMAIL
+    )
+    NOTIFICATION_MAGIC_LINK_MAGIC_LINK_URL = (
+        CommonConfig.NOTIFICATION_MAGIC_LINK_MAGIC_LINK_URL
+    )
+    NOTIFICATION_MAGIC_LINK_REQUEST_NEW_EMAIL_URL = (
+        CommonConfig.NOTIFICATION_MAGIC_LINK_REQUEST_NEW_EMAIL_URL
+    )
+    NOTIFICATION_MAGIC_LINK_FUND_NAME = (
+        CommonConfig.NOTIFICATION_MAGIC_LINK_FUND_NAME
```
Comment (gidsg) [inline config/envs/default.py:108]: Should this come from the fund store API rather than config?

File: frontend/magic_links/routes.py [hunk 5]
```diff
@@ -52,8 +53,10 @@ def new():
     Returns a page containing a single question requesting the
     users email address.
     """
-    fund_id = request.args.get("fund_id")
+    # Default to COF while we only have one fund
+    fund_id = request.args.get("fund_id", Config.FUND_ID_COF)
     round_id = request.args.get("round_id")
+    # current_app.logger.debug("Fund ID:", fund_id, "round ID:", round_id)
```
Comment (gidsg) [inline frontend/magic_links/routes.py:59]: Can re move this as it's commented out?

File: config/envs/default.py (hunk 4 above)
Comment (srh-sloan) [reply config/envs/default.py:108]: It's not the name itself, but the name of the field for fund name in the notification json. So the literal value is just `fund_name`

File: requirements.txt [hunk 6]
```diff
@@ -79,7 +79,7 @@ flask-talisman==0.8.1
     # via -r requirements.in
 flask-wtf==1.0.0
     # via -r requirements.in
-funding-service-design-utils @ https://github.com/communitiesuk/funding-service-design-utils/archive/refs/tags/0.0.19.tar.gz
+funding-service-design-utils @ git+https://github.com/communitiesuk/funding-service-design-utils@bau-notify-fields
```
Comment (gidsg) [inline requirements.txt:82]: Can we change this to a released version?

Comment (srh-sloan) [reply requirements.txt:82]: Will do once I've merged this one https://github.com/communitiesuk/funding-service-design-utils/pull/23

**Review by gidsg — APPROVED** [review]

**Pull request merged**